server/registry.go

```diff
@@ -147,6 +144,8 @@ func createMultiMimeTypeFn(handlersByMimeType map[handlerDTOMimeTypeKey]*handler
 		if contentType == "" {
 			contentType = "*/*"
 		}
+		// get rid of extra annotations - i.e. ;charset=utf-8 or ;boundary=----
```

Comment:
this is reason why yeti was not functioning. Request from auth0 is shaped like this:
![image](https://user-images.githubusercontent.com/113366280/227514434-fa29ed2a-db86-4a36-86e7-d64918efd7ae.png)

and the extra encoding information in Content-Type resulted in this logic not being able to match any handler and returning an error.